patches/openwrt/0008-ath10k-firmware-bump-firmware-for-wave-1-chips.patch

```diff
@@ -0,0 +1,38 @@
+From: David Bauer <mail@david-bauer.net>
+Date: Thu, 26 Sep 2019 18:23:12 +0200
+Subject: ath10k-firmware: bump firmware for wave 1 chips
```

Comment:
This patch should be pushed upstream.

Reply:
this commit was going into the wrong branch, i will remove it. We are currently bisecting our ath10k issues with newer firmware.